### DISCUSSION TITLE
ubuntu22.04での起動

### DISCUSSION THREAD

**Discussion by yamazak** [pf-robotics/kachaka-api#58] [Q&A]
dockerのインストールまでを済まし、
./start_bridge.sh カチャかのIP　とすると以下のように出ます。
permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json?all=1&filters=%7B%22label%22%3A%7B%22com.docker.compose.config-hash%22%3Atrue%2C%22com.docker.compose.project%3Dros2_bridge%22%3Atrue%7D%7D": dial unix /var/run/docker.sock: connect: permission denied

sudoで実行すると起動はできるのですが、
ros2_bridge-ros2_bridge-1  | [component_container_mt-1] [ERROR] [1695365183.261231269] [kachaka.static_tf]: 'static_tf' error #12: 
が延々と出てきて、rviz2 -d kachaka.rviz　を起動してもトピックが受け取れていないようです。

sudo gpasswd -a ユーザー名 docker　としてみましたが現象は変わりませんでした。
解決策がありましたらご教示ください。よろしくおねがいします。

**Comment by youtalk-pfr** [comment]
すみません、ご投稿に気付くのが大変遅くなってしまいました。
Dockerのインストールは https://docs.docker.com/engine/install/ubuntu/ に従った最新版でしょうか？

また、 https://docs.docker.com/engine/install/linux-postinstall/ に従って
`sudo usermod -aG docker $USER`
をお試しください。その後、ログアウト、再ログインが必要です。再起動でもよいです。
